bootloaders/encrypted/aes.S

```diff
@@ -1,45 +1,31 @@
+/* MEMORY LAYOUT ASSUMPTIONS
```

Comment:
Is this `aes.S` the same as the one in https://github.com/raspberrypi/picotool/pull/207 ?
(And if so, do we have any mechanism for ensuring that they stay in sync?)

Reply:
If the picotool CI is now also building pico-examples, perhaps the picotool CI could also check that these files in the two repos match? (or perhaps it's very unlikely that this file will change in future, so perhaps that'd be overkill)

Reply:
That sounds like a good idea, I will add that